Dockerfile.exec-centos5.freesurferbuild-centos4

```diff
@@ -38,7 +38,9 @@ ENV FREESURFER_HOME=/usr/local/src/freesurfer
 
 
 # Add the personal license text file to the Freesurfer directory
-# Example to add the license text - ADD ./license.txt /usr/local/src/freesurfer/license.txt
+# Example to add the license text - 
+ADD ./license.txt /usr/local/src/freesurfer/license.txt
```

Comment:
The license is supposed to be a parameter of the plugin, i.e., it should be added to the JSON descriptor. Otherwise, anyone could use Freesurfer without a license using this container.